pkg/crd/kubermatic/v1/datacenter.go

```diff
@@ -416,3 +418,10 @@ type SeedMLASettings struct {
 	// Optional: UserClusterMLAEnabled controls whether the user cluster MLA (Monitoring, Logging & Alerting) stack is enabled in the seed.
 	UserClusterMLAEnabled bool `json:"user_cluster_mla_enabled,omitempty"`
 }
+
+// Metering contains all the configurations for the metering tool.
+type MeteringConfigurations struct {
+	Enabled          bool   `json:"enabled"`
+	StorageClassName string `json:"storageClassName"`
+	StorageSize      string `json:"storageSize"`
```

Comment:
Can you add a doc comment to each field to describe their default values? Maybe even, when you extend the example-yaml-generator, include the default values there so it' in the generated YAML as the default value, WDYT?

Reply:
Yeah that sound good, I will add a comment as well.